_layouts/documentation.html

```diff
@@ -696,6 +696,10 @@
                             <li
                             {% if page.menu == 'firebird' %} class="nav--vertical__active" {% endif %}>
                             <a href="/documentation/database/firebird">Firebird</a></li>
+
+                            <li
+                            {% if page.menu == 'cloudspanner' %} class="nav--vertical__active" {% endif %}>
```

Comment:
`cloudspanner` should be `cloud-spanner` - it has to match the value of `menu` in the front-matter on `cloud-spanner.md`

Reply:
Fixed below